BUILD

```diff
@@ -23,6 +23,10 @@ release(
         "@onedal//cpp/daal:core_static",
         "@onedal//cpp/daal:thread_static",
         "@onedal//cpp/daal:sequential_static",
+        "@onedal//cpp/daal:core_dynamic",
+        "@onedal//cpp/daal:thread_dynamic",
+        "@onedal//cpp/daal:sequential_dynamic",
         "@onedal//cpp/oneapi/dal:static",
+        "@onedal//cpp/oneapi/dal:dynamic",
```

Comment:
Does it mean we don't want to have several static/dynamic libs of oneDAL (thread/sequential)?

Reply:
We already have `onedal_thread.so`/`onedal_sequential.so` as part of the old DAAL build. 

See lines 
```bazel
"@onedal//cpp/daal:thread_static",
"@onedal//cpp/daal:sequential_static",
"@onedal//cpp/daal:thread_dynamic",
"@onedal//cpp/daal:sequential_dynamic",
```
The libraries that correspond to these targets are named `onedal_thread/sequential.so/a`.

Reply:
I see that we have this for DAAL. Does it mean that all the kernels we develop in new interfaces will be placed in `onedal_thread/sequential.so/a` alongside with old kernels? How this will be organized?

Reply:
There is no kernels inside `onedal_thread` or `onedal_sequential`, this is abstraction layer over threading routines.